src/clang.rs

```diff
@@ -69,7 +69,11 @@ impl Cursor {
 
     /// Get the mangled name of this cursor's referent.
     pub fn mangling(&self) -> String {
-        unsafe { cxstring_into_string(clang_Cursor_getMangling(self.x)) }
+        if clang_Cursor_getMangling::is_loaded() {
+            unsafe { cxstring_into_string(clang_Cursor_getMangling(self.x)) }
+        } else {
+            self.display_name()
```

Comment:
Well, this is wrong for C++, but I guess... Maybe add a `warn!` here or something?

Also, are you sure you want `display_name` and not `spelling`?